test/parallel/test-buffer-alloc.js

```diff
@@ -1,15 +1,11 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
+const common = require('../common');
+const assert = require('assert');
 
-var Buffer = require('buffer').Buffer;
+const Buffer = require('buffer').Buffer;
+const SlowBuffer = require('buffer').SlowBuffer;
 
-// counter to ensure unique value is always copied
-var cntr = 0;
-
-var b = Buffer.allocUnsafe(1024);
-
-console.log('b.length == %d', b.length);
+const b = Buffer.allocUnsafe(1024);
 assert.strictEqual(1024, b.length);
```

Comment:
(And yes, I'm going for the most useless nit I can imagine here. Like I said, feel free to ignore.)